tensorflow_encrypted/protocol/pond.py

```diff
@@ -97,7 +97,7 @@ def define_public_variable(self, initial_value, apply_encoding=True, name=None):
         _initializers.append(x.initializer)
         return x
 
-    def define_private_variable(self, initial_value, apply_encoding=True, name=None):
+    def define_private_variable(self, initial_value, apply_encoding=True, name=None) -> PondPrivateTensor:
```

Comment:
I think you want this to be `PondPrivateVariable`, otherwise you might run into problems when trying to assign to it later!